src/pages/index.js

```diff
@@ -88,19 +130,37 @@ const addProfileFormValidator = new FormValidator(
 
 addProfileFormValidator.enableValidation();
 
+const changeProfilePictureFormValidator = new FormValidator(
+  validationConfig,
+  profilePictureModal
+);
+
+changeProfilePictureFormValidator.enableValidation();
+
 /* -------------------------------- USER INFO ------------------------------- */
 const userProfile = new UserInfo(profileTitle, profileDescription);
 
 /* --------------------------------- #MODALS# --------------------------------- */
+function ButtonLoading() {
```

Comment:
Every variable in the project should be in `camelCase` (in **lower case** if it's not a class or component)